qa/acceptance-tests/src/test/java/io/camunda/it/cluster/ClusterPurgeMultiDbIT.java

```diff
@@ -204,20 +211,16 @@ void shouldPurgeProcessCache() {
                   .satisfies(items -> Assertions.assertThat(items).hasSize(1));
             });
 
-    // Query API so cache is filled - we don't care about the result
-    client
-        .newFlownodeInstanceSearchRequest()
-        .filter((f) -> f.processInstanceKey(processInstanceKey1))
-        .sort(sort -> sort.startDate().asc())
-        .send()
-        .join()
-        .items();
-
     // WHEN
     final var planChangeResponse = actuator.purge(false);
 
     // THEN
     assertThatChangesAreApplied(planChangeResponse);
+    try {
+      Thread.sleep(500);
+    } catch (final InterruptedException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
:x: I would really try and avoid adding `Thread.sleep`, since we never know if it's enough, and will inevitably lead to flakiness. Let's try forcing a refresh after truncation and see if it's still flaky.